Geometry/VeryForwardGeometryBuilder/src/DDDTotemRPConstruction.cc

```diff
@@ -29,61 +31,78 @@ DDDTotemRPContruction::DDDTotemRPContruction()
 
 const DetGeomDesc* DDDTotemRPContruction::construct(const DDCompactView* cpv)
 {
-	using namespace std;
+  // create DDFilteredView and apply the filter
+  DDFilteredView fv(*cpv);
```

Comment:
9X version actually has a filter here
```C++
 DDPassAllFilter filter;
 DDFilteredView fv(*cpv, filter);
```
should it be added here as well or should the comment be updated ?